snc.sh

```diff
@@ -388,6 +388,11 @@ ${YQ} write --inplace ${INSTALL_DIR}/install-config.yaml sshKey "$(cat id_rsa_cr
 # Create the manifests using the INSTALL_DIR
 ${OPENSHIFT_INSTALL} --dir ${INSTALL_DIR} create manifests || exit 1
 
+# Use a fixed node name
+OLD_INFRA_NAME=$(${YQ} read ${INSTALL_DIR}/manifests/cluster-infrastructure-02-config.yml 'status.infrastructureName')
+NEW_INFRA_NAME=$(${YQ} read install-config.yaml metadata.name)-abcde
```

Comment:
for `NEW_INFRA_NAME` can be just `${CRC_VM_NAME}-abcde` instead reading the install config again because that what we do as part install-config https://github.com/code-ready/snc/blob/master/snc.sh#L382